docs/csharp/language-reference/proposals/toc.yml

```diff
@@ -88,6 +88,8 @@
       href: ../../../../_csharplang/proposals/csharp-9.0/records.md
     - name: Top-level statements
       href: ../../../../_csharplang/proposals/csharp-9.0/top-level-statements.md
+    - name: Nullable reference types - specification
```

Comment:
Why does this appear twice?

Reply:
Because the feature was added in C# 8.0, and is now being updated for C# 9.0. I think it will help findability. On the other hand, do you think it will be confusing to appear twice?

Reply:
I think it's probably fine. I did notice in the preview that "proposal" should be plural in the top level node for 8 and 9.

Reply:
@BillWagner Not sure if you saw the previous comment.